js/ui/handler/touch_zoom_rotate.js

```diff
@@ -41,13 +41,23 @@ class TouchZoomRotateHandler {
     /**
      * Enables the "pinch to rotate and zoom" interaction.
      *
+     * @param {Object} options
+     * @param {string} options.around If "center" is passed, map will zoom around the center
+     *
      * @example
      *   map.touchZoomRotate.enable();
+     * @example
+     *   map.touchZoomRotate.enable({ around: 'center' });
      */
-    enable() {
+    enable(options = {}) {
```

Comment:
I would just write `this._aroundCenter = options && options.around === 'center'`.